.env.example

```
@@ -1 +1,6 @@
 PORT=3333
+DB_HOST=
+DB_PORT=
+DB_USER=
+DB_PWD=
+DB_NAME=
```

```suggestion
DB_HOST=localhost
DB_PORT=3306
DB_USER=example-user
DB_PWD=example-password
DB_NAME=sexta_eh_nois
```
Você pode adicionar os valores default para indicar qual valor alterar para quem forkar o projeto.